components/alert/check-props.js

```diff
@@ -3,51 +3,55 @@
 /* eslint-disable import/no-mutable-exports */
 
 import sunsetProperty from '../../utilities/warning/sunset-property';
+import getComponentDocFn from '../../utilities/get-component-doc';
 
 let checkProps = function () {};
 
 if (process.env.NODE_ENV !== 'production') {
 	checkProps = function (COMPONENT, props) {
-		/* eslint-disable max-len */
-		sunsetProperty(
-			COMPONENT,
-			props.iconCategory,
-			'iconCategory',
-			'Use `Icon` instead.'
-		);
-		sunsetProperty(
-			COMPONENT,
-			props.iconName,
-			'iconName',
-			'Use `Icon` instead.'
-		);
+		import('./docs.json').then((jsonDoc) => {
+			const createDocUrl = getComponentDocFn(jsonDoc);
+			/* eslint-disable max-len */
+			sunsetProperty(
+				COMPONENT,
+				props.iconCategory,
+				'iconCategory',
+				`Use \`Icon\` instead.${createDocUrl('icon')}`
```

Comment:
`createDocUrl()` optionally takes in a prop and returns a message that direct users to the doc site of the component and possibly a prop.